src/vega.schema.ts

```diff
@@ -292,6 +292,8 @@ export interface VgAxisBase {
    *
    * __Default value:__  derived from [axis config](config.html#axis-config)'s `tickSize` (`6` by default).
    *
+   * __Default value:__ `2/3*rangeStep` (This will provide offset between band equals to the width of the tick.)
```

Comment:
I added the default to the wrong place. And now we don't have `tickSize` as a tick config anymore. it's renamed to `bandSize `